README.md

```diff
@@ -164,6 +166,7 @@ The output is going to look like the following:
 [+]       Based on:...2 features:
```

Comment:
Did you updated the sample output manually or re-created it with the demo tool? I think the tool should now print "3 features".